Rakefile

```diff
@@ -104,14 +87,7 @@ namespace :breathe do
 
     people_to_sync.each { |person| person.sync_breathe_to_productive(after: earliest_date) }
   rescue => e
```

Comment:
Previously (before we added the rescue and Slack alerting) the error was allowed to stop the execution of the task, which is why we were re-raising it after alerting to Slack. Is it intentional that you're not re-raising the error after rescuing it? Or will Rollbar do that automatically?

Reply:
No that wasn't an intentional change, and I believe Rollbar would otherwise swallow it. I've put it back in now.